src/urllib3/response.py

```diff
@@ -90,6 +95,21 @@ def decompress(self, data):
             self._obj = zlib.decompressobj(16 + zlib.MAX_WBITS)
 
 
+if brotli is not None:
+    class BrotliDecoder(object):
+        def __init__(self):
+            self._obj = brotli.Decompressor()
+
+        def __getattr__(self, name):
+            return getattr(self._obj, name)
+
+        def decompress(self, data):
+            try:
+                return self._obj.decompress(data)
+            except brotli.Error as err:
```

Comment:
No need to re-raise here if we're adding `brotli.Error` below?

Reply:
Good catch!